src/web/security/core/src/main/java/org/geoserver/security/web/role/RolePaletteFormComponent.html

```diff
@@ -4,27 +4,25 @@
    xmlns:wicket="http://wicket.apache.org/">
 <body>
 <wicket:panel>
- <ul>
-  <li>
-   <fieldset>
-      <!-- <legend><span><wicket:message key="roles">roles</wicket:message></span></legend> -->
-      <legend><span wicket:id ="roles"></span></legend>
-      <ul>
-         <li>
-           <wicket:child></wicket:child>
-         </li>
-         <li>
-           <div wicket:id="palette"></div>
-         </li>
-         <li>
-           <a class="add-link" href="#" wicket:id="addRole">
-             <wicket:message key="addRole">addRole</wicket:message>
-           </a>
-         </li>
-      </ul>
-   </fieldset>
-  </li>
- </ul>
+
+  <fieldset>
+    <!-- <legend><span><wicket:message key="roles">roles</wicket:message></span></legend> -->
```

Comment:
I see some commented out html has been left behind here, I think you can delete this as part of the cleanup

Reply:
Ok, I will clean it. Didn't know what to do with it, that's why I left it there.